src/store/actions/actions-model.ts

```diff
@@ -124,3 +124,7 @@ export const setIgnoreChanges: ActionIF = ({ commit }, ignoreChanges): void => {
 export const setHaveChanges: ActionIF = ({ commit }, haveChanges): void => {
   commit('mutateHaveChanges', haveChanges)
 }
+
+export const setValidateSteps: ActionIF = ({ commit }, validate): void => {
```

Comment:
Does this have a type?

Reply:
It does, we haven't defined any types here though. This means changing every action?

Reply:
Prob just change this one unless you have extra time (haha).